src/js/angular/repositories/ontop-repo.directive.js

```diff
@@ -156,13 +156,24 @@ function ontopRepoDirective($uibModal, RepositoriesRestService, toastr, Upload,
          * @param {OntopFileInfo} ontopFileInfo
          */
         $scope.editFile = (ontopFileInfo) => {
+
+            let fileName = $scope.repositoryInfo.params[ontopFileInfo.type].label;
+            if (fileName.indexOf(' file')) {
```

Comment:
This check to test the end of the string and remove the word 'file' is not correct. IMO it should be like `if (fileName.indexOf(' file') === fileName.length - 5) {`

Reply:
Doesn't say it removes it from the end.

Reply:
> 

Maybe because there can be names without "file" in them.

Reply:
Because file is not on French :(. I Remove it from the label and translate it in the label.